notebooks/huggingface_notebook.ipynb

```diff
@@ -0,0 +1,1349 @@
+{
```

Comment:
Please remove the warnings and update `notebooks/README.md`

Reply:
Re-ran notebook with updated display.